backend/server.ts

```diff
@@ -90,4 +90,14 @@ app.listen(port, () => {
     console.log(`server started at http://localhost:${port}`);
 });
 
+//GET package download
+app.get('/package/download', async (req, res) => {
+    try {
+        await apiPackage.getPackageDownload(req, res);
+    } catch (error) {
+        res.status(500).send('Internal Server Error');
+    }
+});
 
+//PUT package update
+app.put('/packages/update', apiPackage.updatePackage);
```

Comment:
Need to follow format of other requests. Use (req, res) even if the updatePackage function doesn't use it.